src/App.tsx

```diff
@@ -50,24 +133,27 @@ export const App: React.FC = () => {
 
               <div className="dropdown-item" data-cy="suggestion-item">
                 <p className="has-text-danger">Elisabeth Hercke</p>
+              </div> */}
               </div>
             </div>
-          </div>
+          )}
         </div>
 
-        <div
-          className="
+        {showMessageRef.current && (
```

Comment:
you use it for conditional rendering - so it should be state, because the ref value change doesn't cause rerender
Even if other state values change at the same time and it will work you shouldn't do it like this